RELEASE.md

```diff
@@ -0,0 +1,20 @@
+# Plugin broker release process
+
+Che plugin broker releases are built automatically from webhooks on the `release` branch and uses the version specified in `VERSION` to tag the built images. To create a new release:
+
+1. Create a `major.minor.x` branch if necessary (e.g. `v3.2.x`)
+2. Update `VERSION` file to refer to the new version to be released, with bugfix version number included (e.g. `v3.2.0`)
+    - Note the convention for the Che plugin broker is to prefix release versions with `v`, i.e. `v3.2.0` instead of `3.2.0`.
```

Comment:
Because... daring to be different? :P

Reply:
but it's cool. some projects use a x.y.z version as a branch. A branch! https://github.com/che-incubator/chectl/tree/7.7.1 << what is that madness?

Reply:
We mostly keep doing it because that's how we started doing it. `v3.0.0` was released so it wouldn't make sense to switch conventions for `3.1.0`, etc.

Reply:
Well, I believe leading v is a requirement from Golang.
There are workarounds we could use if case versioning without `v` but they are workarounds.

>  valid semver tags (including the required leading v)
from https://github.com/golang/go/wiki/Modules#can-a-module-consume-a-package-that-has-not-opted-in-to-modules